qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -270,15 +276,23 @@ def observable(me_table, init_term=0, mapping="jordan_wigner", wires=None):
 
     **Example**
 
-    >>> table = np.array([[0.0, 0.0, 0.4], [1.0, 1.0, -0.5], [1.0, 0.0, 0.0]])
-    >>> print(observable(table, init_term=1 / 4, mapping="bravyi_kitaev"))
-    (0.2) [I0]
-    + (-0.2) [Z0]
-    + (0.25) [Z0 Z1]
-    >>> print(observable(table, init_term=1 / 4, mapping="bravyi_kitaev", wires=['w0','w1']))
-    (0.2) [Iw0]
-    + (-0.2) [Zw0]
-    + (0.25) [Zw0 Zw1]
+    >>> t = np.array([[0., 0., 0.5], [1.0, 1.0, -0.5], [1.0, 0., 0.]])
+    >>> v = np.array([[ 0., 0., 0., 0., 0.25], [ 0., 1., 1., 0., -0.25], [ 1., 0., 0., 1., -0.5]])
+    >>> me_tables = []
+    >>> me_tables.append(t)
+    >>> me_tables.append(v)
+
+    >>> print(observable(me_tables, init_term=1/4, mapping="bravyi_kitaev"))
```

Comment:
```suggestion
    >>> print(observable(me_tables, init_term=1/4, mapping="bravyi_kitaev"))
```
imo it looks and reads better not having the gap